osier/models/capacity_expansion.py

```diff
@@ -123,6 +125,25 @@ def __init__(self,
                          xl=0.0,
                          xu=1.0,
                          **kwargs)
+        
+    def pprint(self):
```

Comment:
This is not super readable. It looks like a typo. Maybe something like `print_problem_formulation`?